src/devices/bus/coco/coco_pak.h

```diff
@@ -68,9 +68,40 @@ class coco_pak_banked_device : public coco_pak_device
 	u8 m_pos;
 };
 
+// ======================> coco_pak_rmulticart_device
+// Ramoth Dragon/CoCo Multicart.
+
+class coco_pak_rmulticart_device : public coco_pak_device
+{
+public:
+	// construction/destruction
+	coco_pak_rmulticart_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock);
+
+protected:
+	coco_pak_rmulticart_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock);
+
+	// device-level overrides
+	virtual const tiny_rom_entry *device_rom_region() const override;
+	virtual void device_start() override;
+	virtual void device_reset() override;
+	virtual u8 *get_cart_base() override;
+	virtual u32 get_cart_size() override;
+	virtual u8 cts_read(offs_t offset) override;
+	virtual u8 scs_read(offs_t offset) override;
+	virtual void scs_write(offs_t offset, u8 data) override;
+
+	void SetCART();
+private:
+	u8 Page;		// Page register, at $FF50
+	u8 CTRL;		// Control register, at $FF52
+	u16 PageSize;	// Page size in bytes
+	u8 PageMask;	// Page mask
```

Comment:
Please follow conventions for case in names.  We use `LlamaCase` for template parameter names and `SCREAMING_SNAKE_CASE` for macros/constants.  We use `snake_case` for members, with an optional `m_` prefix for data members.  The other classes in this file use the prefix, so you should follow that convention.